schema_version: 1.2.0
id: CVE-2016-4445
aliases: []
modified: 2017-04-17T13:16Z
published: 2017-04-11T18:59Z
details: The fix_lookup_id function in sealert in setroubleshoot before 3.2.23 allows
  local users to execute arbitrary commands as root by triggering an SELinux denial
  with a crafted file name, related to executing external commands with the commands.getstatusoutput
  function.
severity:
- type: CVSS_V3
  score: 7.0
- type: CVSS_V2
  score: 6.9
affected:
- package:
    name: enterprise_linux_hpc_node
    cpe: cpe:2.3:o:redhat:enterprise_linux_hpc_node
- ranges:
  - type: GIT
    repo: https://github.com/fedora-selinux/setroubleshoot
    events:
    - introduced: '0'
    - fixed: 2d12677629ca319310f6263688bb1b7f676c01b7
- versions:
  - '7.0'
- database_specific:
    CWE:
    - CWE-77
    CVSS:
      Score: 7.0
      Severity: HIGH
      Code: CVSS:3.0/AV:L/AC:H/PR:L/UI:N/S:U/C:H/I:H/A:H
references:
- type: FIX
  url: https://github.com/fedora-selinux/setroubleshoot/commit/2d12677629ca319310f6263688bb1b7f676c01b7
- type: WEB
  url: http://www.securitytracker.com/id/1036144
- type: WEB
  url: http://seclists.org/oss-sec/2016/q2/575
- type: WEB
  url: http://www.securityfocus.com/bid/91430
- type: WEB
  url: https://rhn.redhat.com/errata/RHSA-2016-1267.html
- type: REPORT
  url: https://bugzilla.redhat.com/show_bug.cgi?id=1339183
